app/views/projects/index.html.erb

```diff
@@ -0,0 +1 @@
+<%= render "layouts/sidebar.html.erb" %>
```

Comment:
Does this mean I should I see the sidebar when I go to /projects ? Because I don't :/